source/ProSymbolEditor/ViewModels/MilitarySymbolDockpaneViewModel.cs

```diff
@@ -804,77 +947,122 @@ private void CopyImageToClipboard(object parameter)
             }
         }
 
-        private void LoadSymbolFile(object parameter)
+        private void LoadSymbolFile(bool isEditSymbol)
```

Comment:
Sorry, I don't quite understand this method/name - can you change to a more meaningful name or add some comments about what method generally does (ex: sets symbol model state from...? and perhaps how edit case is different).